lib/cdo/properties.rb

```diff
@@ -54,16 +54,28 @@ def self.get_user_metrics
 end
 
 def fetch_metrics
-  # Include stale default values as of 2017-06-21 so we never show 0. These
+  # Include stale default values as of 2017-06-21 (project count from 2019-1-20) so we never show 0. These
   # would be used, for example, if the DB is unavailable.
   Properties.get(:metrics) || {
     'created_at' => '2017-06-21T14:46:25+00:00',
     'created_on' => 'created_on"=>"2017-06-21',
     'petition_signatures' => 1_774_817,
-    'lines_of_code' => 21_238_497_830
+    'lines_of_code' => 21_238_497_830,
+    'project_count' => 35_000_000
   }
 end
 
+# Puts project count into format of XX million for display
+def round_project_count_to_million(project_count)
+  # check if number is bigger than previous count
+  existing_project_count = Properties.get(:metrics)['project_count'] || 0
```

Comment:
this is the same point that @breville mentioned in [this comment](https://github.com/code-dot-org/code-dot-org/pull/26702#discussion_r250371107), but `Properties.get(:metrics)['project_count']` and the `project_count` argument will be equal, since we've already saved the project_count metric to the database and fetched it in the view that calls this method. so we can remove `existing_project_count` since it will always equal `project_count`.

if we have to make sure the new project count is greater than the existing project count, we will have to do that in the cron job before we save the new project count and wipe away the existing project count. however, i'm not sure we actually care that the project count is increasing -- if, for whatever reason, our project count decreases, shouldn't we reflect the correct number on our website? (tagging @poorvasingal as she has more context for this)